tests/e2e/conftest.py

```diff
@@ -0,0 +1,64 @@
+import pytest
+import tempfile
+from acuvity import Acuvity, Security
+import os
+
+SUPPORTED_PLATFORMS = ["dev", "prod", "pre_prod"]
+
+
+def pytest_addoption(parser):
+    parser.addoption(
+        "--platform",
+        action="store",
+        default="",
+        type=str,
+        help="Specify platform to run against",
+    )
+
+
+def pytest_configure(config):
+    config.acuvity_platform = config.getoption("--platform")
+
+
+@pytest.fixture(scope="class")
+def get_token(request):
+    platform = request.config.acuvity_platform
+    if platform:
+        assert platform in SUPPORTED_PLATFORMS
+        token = os.getenv(f"APPS_{platform.upper()}_TOKEN", "")
+        assert token, f"APPS_{platform.upper()}_TOKEN env variable missing"
+    else:
+        token = os.getenv(f"ACUVITY_TOKEN", "")
+        assert token, "ACUVITY_TOKEN env variable missing"
+    return token
+
+
+@pytest.fixture(scope="class")
+def init_apex(get_token):
+    with Acuvity(
+        security=Security(token=get_token),
+    ) as acuvity:
```

Comment:
I'm not a big fan of this context manager that Speakeasy generates (because internally that is a lot of initializing/deinitializing all the time which is increased for us because we have custom code in that code path). And, unfortunately, it's also all over the docs. I'm even a bit surprised that this actually works properly 😅 

Was there a specific reason why you used this here? I guess because that's what the README in the docs suggests to use? If we could simply use `c = Acuvity(security=Security(token=get_token))`, I would prefer that.